Cargo.toml

```diff
@@ -146,3 +146,11 @@ codegen-units = 1
 lto = true
 opt-level = 3
 panic = "abort"
+
+[lints.rust]
+unexpected_cfgs = { level = "allow", check-cfg = [
+    # Legacy code
+    'cfg(portable)',
```

Comment:
I am not too sure about the `portable` config (i.e. is the config supposed to be "hidden", or is this just remainder from previous code), so I have elected to ignore the warning about it here,.